package.json

```diff
@@ -8,10 +8,35 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "react-dev": "webpack -d --watch",
+    "server-dev": "nodemon server/index.js",
```

Comment:
`build` and `start` are more common terms than `react-dev` and `server-dev`.

Also, I would recommend alphabetizing the key-value pairs of your object unless you have a compelling reason to do so otherwise.